packages/preservation_spine/cli.py

```
@@ -0,0 +1,712 @@
+
+    return parser
+
+
+def main(argv: Sequence[str] | None = None) -> int:
+    namespace = build_parser().parse_args(list(argv) if argv is not None else None)
+    try:
+        return int(namespace.handler(namespace))
+    except (CapsuleVerificationError, CheckpointIntegrityError, ValueError) as exc:
+        _emit_error(str(exc))
+        return 1
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)